Bottom: 6e003f93bc1849300ec7e090ad13d770b5a7cdc9
Top:    364b974c07962e4a04e43f6dea6416eab8d8176d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-20 23:18:11 +0000

Put back layer opacity to its old behaviour

The exact details of this setting aren't right yet,
so revert them so I can build a release.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 119f421..b825182 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1786,7 +1786,7 @@ ghid_draw_everything (BoxType *drawn_area)
   }
 
   /* draw all layers in layerstack order */
-#define FADE_FACTOR 0.8
+#define FADE_FACTOR 1
   number_phys_on_top = max_phys_group - min_phys_group;
   for (i = ngroups - 1; i >= 0; i--) {
     bool is_this_physical = drawn_groups[i] >= min_phys_group &&
